include/gatekeeper_net.h

```diff
@@ -21,6 +21,9 @@
 
 #include <stdint.h>
 
+/* The RSS hash uses a secret key of length 40 bytes. */
```

Comment:
Suggestion to improve this comment:

`/* Size of the secret key of the RSS hash. */`